### PR TITLE
[Localisation] [JA] Japanese backlog localisation to json

### PR DESCRIPTION
**Pull request by AsdarDevelops**
Meant to close #3555 . I re-added the .ts as .json .

### CODE REVIEW

**Review by flx-sta — APPROVED** [review]
I don't speak japense but it looks like japanese

**Review by NicusPulcis — DISMISSED** [review]
as part of the italian translation team, this looks japanese enough to me

**Review by Adri1 — APPROVED** [review]

File: src/locales/ja/menu-ui-handler.json [hunk 1]
```diff
@@ -1,19 +1,27 @@
 {
-  "GAME_SETTINGS": "せってい",
-  "ACHIEVEMENTS": "じっせき",
-  "STATS": "とうけい",
-  "RUN_HISTORY":"ラン履歴",
+  "GAME_SETTINGS": "設定",
+  "ACHIEVEMENTS": "実績",
+  "STATS": "統計",
+  "VOUCHERS": "クーポン",
```
Comment (frutescens) [inline src/locales/ja/menu-ui-handler.json:5]: This should be removed.
```suggestion
```

Comment (AsdarDevelops) [reply src/locales/ja/menu-ui-handler.json:5]: Correct

**Review by frutescens — APPROVED** [review]

**Review by DayKev — APPROVED** [review]

**Review by flx-sta — APPROVED** [review]

**Pull request merged**